airbyte-ci/connectors/live-tests/tests/backends/test_file_backend.py

```diff
@@ -16,7 +16,6 @@
 from live_tests.commons.backends import FileBackend
 
 
-@pytest.mark.asyncio
```

Comment:
❓ curious, why did we need to un-asyncify things?

Reply:
Because I don't get why they were originally asyncified 😄 There's no async logic on the `FileBackend`, so I don't think it should be using `async/await`.

Reply:
Lol, fair enough!